contrib/migration/migrate-compose-file-v1-to-v2.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+"""
+Migrate a Compose file from the V1 format in Compose 1.5 to the V2 format
+supported by Compose 1.6+
+"""
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import argparse
+import sys
+
+import ruamel.yaml
+
+
+def migrate(content):
+    data = ruamel.yaml.load(content, ruamel.yaml.RoundTripLoader)
+
+    service_names = data.keys()
+    for name, service in data.items():
+        # remove links and external links
+        service.pop('links', None)
+        service.pop('external_links', None)
```

Comment:
Should we perhaps show a warning if the user has defined anything we can't seamlessly convert? No-alias `links` entries are fine and can be discarded, but anything else is probably going to break their app if it's missing.